src/bin/console/console.cpp

```diff
@@ -675,8 +675,8 @@ int Console::print_transaction_info(const std::string& input) {
   }
```

Comment:
I think it’s a good idea to try to better distinguish these too commit IDs. Perhaps it’s just me, but I don’t find `as_of_commit_id` really self-explanatory. the last commit ID of the transaction context represents a certain snapshot or view into the database so my idea would be something like `snapshot_commit_id`.

Reply:
`snapshot_commit_id` works for me too. The first underscore in `as_of_commit_id` kinda hurt the name, anyway